src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/contexts/SymptomContext.java

```diff
@@ -19,23 +19,11 @@
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.GenericContext;
 
 public class SymptomContext extends GenericContext {
```

Comment:
This class is more like a placeholder because what needs to be defines here is still not very clear. So let's keep this as is.

Reply:
I've only taken out common parts, if we need to change the logic in the future overriding should be possible